github-page/_docs/decorators/prop.md

```diff
@@ -310,16 +316,18 @@ class Enumed {
 ```
 
 Known-Issues:
-- Babel dosnt set the type for enums correctly, the need to be set manually with [the `type` option](#type)
-- If the code got transpiled with `tsc --transpile-only` or `ts-node --transpile-only` then `--transpile-only` neeeds to be removed or the type needs to be set manually with [the `type` option](#type)
+
```

Comment:
why did the following list get spaced out?

Reply:
I have no idea. Must have been VSCode too.

Scott